editor/src/clj/editor/validation.clj

```diff
@@ -87,7 +87,7 @@
                "'%s' must be between %d and %d"
                "'%s' must be between %f and %f")]
     (when (not (<= min v max))
-      (format tmpl name min max))))
+      (util/format* tmpl name min max))))
```

Comment:
`util/format*` uses the `ROOT` locale instead of the system locale to ensure float values have decimal points and not commas.